### ISSUE TITLE
Some keys are not working, dunno how to debug 

### ISSUE THREAD

**Issue by jorgelrezende** [keyzog/kapl#8]
Hello, I've finished to assemble on half of my keyboard, but some keys are not working, my question is, all keys only work when the two halfs are assembled together? or each half should work all keys independently?

**Comment by keyzog** [comment]
Hello! Some buttons are responsible for switching layers. When you click on them, it may seem that nothing happens, because... Most keyboard testing programs/sites cannot detect such keystrokes.

Below is a screenshot with these buttons:
![kapl1](https://github.com/keyzog/kapl/assets/7888825/d2ee8ef6-fa10-4e1f-b1fc-82bab509faaa)

**Comment by jorgelrezende** [comment]
right, but all the esc column is not work here on testing, the 5 key and backspace as well, i will try to  verify with a multimeter if all diodes are working, but i dont know what is happening

**Comment by keyzog** [comment]
If the entire column does not work, it may be due to poor contact on the microcontroller pin. Make sure it is well soldered.
![kapl3](https://github.com/keyzog/kapl/assets/7888825/987815bd-954f-428b-bcac-e63c484099f0)


When one button in col/row does not work, it is most likely due to poor soldering of the button legs. In second place, the problem may be in the wrong direction of the diode next to the button.

you can write to me in telegram and send me a photo of the board
https://t.me/keyzog

**Comment by jorgelrezende** [comment]
Right, i will verify it, thank you for the help
